src/modules/dashboard/components/Dialogs/GnosisControlSafeDialog/TransactionTypesSection/GnosisNFTTransfer.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useMemo } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Avatar from '~core/Avatar';
+import { DialogSection } from '~core/Dialog';
+import SingleUserPicker, {
+  SingleNFTPicker,
+  filterUserSelection,
+} from '~core/SingleUserPicker';
+
+import { useMembersSubscription } from '~data/index';
+
+import { Address } from '~types/index';
+import { NFT } from '~dashboard/Dialogs/GnosisControlSafeDialog';
+
+import styles from './GnosisNFTTransfer.css';
```

Comment:
This style file currently doesn't exist and it's making the dapp not compile :(

Reply:
Morning... I'm sorry & surprised to hear this!!! It was working well for me.....
I dont know how this could happen but i'll put on my Sherlock hat and investigate!

Reply:
My dear Watson.... When you have eliminated the impossible, whatever remains, however improbable, must be the truth. Something went wrong....... 

It was a config issue ;) Fixed now